llpc/patch/llpcPatchNullFragShader.cpp

```diff
@@ -155,13 +155,13 @@ bool PatchNullFragShader::runOnModule(
     auto pInputTy = Type::getFloatTy(*m_pContext);
     std::string importName = LlpcName::InputImportGeneric;
     AddTypeMangling(pInputTy, importArgs, importName);
-    auto pInput = EmitCall(importName, pInputTy, importArgs, NoAttrib, pInsertPos);
+    auto pInput = EmitCall(importName, pInputTy, importArgs, {}, pInsertPos);
```

Comment:
nit: `auto* pInput = ...`

Reply:
That's orthogonal to the change actually happening in that line, so I don't want to fix it in the same change.